doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,131 @@
 Traceability to Reference Model.
 
 <a name="2.3"></a>
-## 2.3 Requirements.
+## 2.3 Containerised Platform Requirements.
+
+The Reference Model (RM) defines the NFVI to consist of the physical resources, virtualised resources and a software managment system.  In the virtualised world, the NFVI Software Management System (Virtualised Infrastructure Manager -- VIM) consists of the Guest Operating System, Hypervisor and, if needed, other software such as libvirt.  And the VIM is responsible for, among others, tenant management, resources management, inventory, scheduling, and access management. Now consider, the containerisation equivalent.
+
+Containerised Infrastructure Manager (CIM): a functional block that is responsible for controlling and managing the containerised NFVI compute, storage and network resources. Note CIM supports the Life Cycle Management of conatinerised and the underlying physical resources.
 
 <a name="2.3.1"></a>
 ### 2.3.1 General
 
 | Ref # | sub-category | Description |
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `req.gen.ost.01` | Open source | The Architecture **must** use Kubernetes APIs.|
-| `req.gen.ost.02` | Open source | The Architecture **must** support dynamic request and configuration of virtual resources (compute, network, storage) through Kubernetes APIs. |
+| `req.gen.k8s.01` | Open source | The Architecture **must** use Kubernetes APIs.|
+| `req.gen.k8s.02` | Open source | The Architecture **must** support dynamic request and configuration of resources (compute, network, storage) through Kubernetes APIs. |
+| `req.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component. |
+| `req.gen.cnt.02` | Cloud nativeness | The Architecture **should** consist of service components implemented as microservices that are individually dynamically scalable. |
```

Comment:
I would add requirement about immutable infrastructure.

Reply:
> I would add requirement about immutable infrastructure.

Was this addressed in the call yesterday?  If not we should bring it up on Friday as I think it should be included (perhaps in the infrastructure section rather than General but definitely somewhere.

Reply:
Added  `req.gen.cnt.03` .